packages/components/src/toggle-control/test/index.js

```diff
@@ -1,52 +0,0 @@
-/**
- * External dependencies
- */
-import renderer from 'react-test-renderer';
-
-/**
- * Internal dependencies
- */
-import ToggleControl from '../';
-
-describe( 'ToggleControl', () => {
-	it( 'triggers change callback with numeric value', () => {
```

Comment:
Curious test name, I don't understand the "numeric value" part

Reply:
Yeah, likely copypasta 🍝